tests/Unit/SpanContextTest.php

```diff
@@ -35,7 +35,7 @@ public function testGetBaggageItemsReturnsExpectedValues()
             self::BAGGAGE_ITEM_KEY => self::BAGGAGE_ITEM_VALUE,
         ]);
 
-        $this->assertEquals(self::BAGGAGE_ITEM_VALUE, $context->getBaggageItem(self::BAGGAGE_ITEM_KEY));
+        $this->assertSame(self::BAGGAGE_ITEM_VALUE, $context->getBaggageItem(self::BAGGAGE_ITEM_KEY));
```

Comment:
Not sure about this change. `assertSame` gives me the feeling I am comparing objects and not the value of the string which is what we aim to test.

Reply:
@jcchavezs 

I think it depends on whether we care that a `string` value is returned, or would be fine, for example if an object were returned that could be casted to `string`. 

Not sure, what do you think?

Personally, I prefer to be as strict as possible, but I'm aware you know much more about this project than I do!

Reply:
I agree with José here; being strict on types in tests when we just want a string-ish value could lead to brittleness later. Probably not in baggage items, but as a general philosophy I'd rather go that way. Kind of like accepting interfaces ... we want to get back something that acts like a string, it shouldn't be important what the actual implementation is.